main.js

```diff
@@ -319,41 +322,76 @@
      * @returns {Promise<Number>} If answer found, return answer.
      */
     function getAnswer_blackxblue() {
-        if (GM_getValue("anime_answer_date") == undefined)
-            if (window.confirm("您選擇了由 blackxblue 小屋獲取答案，是否訂閱 blackxblue？（作答過問題就不會再出現提醒）"))
-                follow("blackxblue");
+        if (GM_getValue("anime_answer_date") === undefined &&
+            window.confirm("您選擇了由 blackxblue 小屋獲取答案，是否訂閱 blackxblue？（作答過問題就不會再出現提醒）")) {
+            follow("blackxblue");
+        }
         return new Promise(function (resolve, reject) {
             GM_xmlhttpRequest({
                 method: "GET",
-                url: "https://home.gamer.com.tw/creation.php?owner=blackxblue",
-                responseType: "text",
+                url: "https://api.gamer.com.tw/home/v2/creation_list.php?owner=blackxblue&row=1",
```

Comment:
I'd prefer to leave this as default and filter out the creation for today's Animad answer to prevent the data provider (blackxblue) additionally have another creation about his life or any other topics.

```suggestion
                url: "https://api.gamer.com.tw/home/v2/creation_list.php?owner=blackxblue",
```

Reply:
but it's fine here to leave the TODO since the function is broken and this question may be pending.